rust/lance/src/dataset/write/commit.rs

```diff
@@ -23,6 +14,14 @@ use crate::{
     session::Session,
     Dataset, Error, Result,
 };
+use lance_core::utils::mask::RowIdTreeMap;
```

Comment:
looks like there are quite a few changes in import order without any actual changes across files, are these unintentional?

Reply:
yes, I found it, that's because I tried tracing in commit, but I changed the decision. It's strange, the format and code style check not recognize it.